packages/web-app-files/src/components/FilesList/ResourceTable.vue

```diff
@@ -682,8 +682,8 @@ export default defineComponent({
        * Triggered when the file row is clicked
        * @property {object} resource The resource for which the event is triggered
        */
-      const resource = data[0]
-      const eventData = data[1]
+      const resource = Array.isArray(data) ? data[0] : data
+      const eventData = Array.isArray(data) ? data[1] : null
```

Comment:
if data != array, eventData is always null?

Sounds strange to me, where is this used and in which cases data is it not of type array of resources?